build.sbt

```diff
@@ -4,9 +4,9 @@ name := "coverage-parser"
 
 version := "1.1.1"
 
-scalaVersion := "2.11.6"
+scalaVersion := "2.11.11"
```

Comment:
The default value should't be the lower one, to make sure the api is compatible with the others?